app/views/AdminPanelView/index.js

```diff
@@ -35,7 +36,10 @@ class AdminPanelView extends React.Component {
 			<SafeAreaView style={[styles.container, { backgroundColor: themes[theme].backgroundColor }]} testID='terms-view'>
 				<StatusBar theme={theme} />
 				<WebView
-					source={{ uri: `${ baseUrl }/admin/info?layout=embedded` }}
+					source={{
+						uri: `${ baseUrl }/admin/info?layout=embedded`,
+						headers: RocketChatSettings.customHeaders
+					}}
```

Comment:
This ins't work on iOS, maybe you can check the [documentation](https://github.com/react-native-community/react-native-webview/blob/master/docs/Guide.md#working-with-custom-headers-sessions-and-cookies) and follow something that you have forgotten.

Reply:
@djorkaeffalexandre I found this [issue](https://github.com/react-native-community/react-native-webview/issues/4), it has still not been addressed. They suggest to [patch](https://github.com/react-native-community/react-native-webview/issues/4#issuecomment-427671845) `RCTWebView.m` as a workaround.